### PR TITLE
Chinese Translation of 1,2 and 9th spec docs which has merged  i18n

### PR DESCRIPTION
**Pull request by KaiwenWang**
This is chinese translation work of 1,2 and 9th spec docs which has merged repo's i18n branch.

Welcome All Reviews.

### CODE REVIEW

File: i18n/zh-Hans/1.purpose_and_goals.md [hunk 1]
```diff
@@ -1,3 +1,34 @@
 # 目标与愿景
 
-// TODO
\ No newline at end of file
+Open Application Model（开放应用模型） 定义了一个标准的、平台无关的方式（规范）来描述云上和边缘的应用。 规范的目标是提供一种通用的方式来描述不依赖于任何特定容器运行时、编排软件、云提供商或硬件配置的应用程序，并为开发和运维（运营）人员定义清晰的角色，同时依然允许规范的不同实现（项目）利用这些实现（项目）自身和它依赖的底层平台所特有的原生API、工具和特性。
```
Comment (wonderflow) [inline i18n/zh-Hans/1.purpose_and_goals.md]: 直接写运维人员叫好了，不用括号运营。

File: i18n/zh-Hans/1.purpose_and_goals.md [hunk 2]
```diff
@@ -1,3 +1,34 @@
...
+
+规范要解决的核心问题是，分布式应用能被如何组织,然后成功地交付给运维（运营）人员。这个核心问题不怎么关乎怎么写代码，而是更侧重如何获取面向（微）服务架构的程序组件，并调优这类应用周边的（研发交付）工作流
```
Comment (wonderflow) [inline i18n/zh-Hans/1.purpose_and_goals.md]:
```suggestion
规范要解决的核心问题是，分布式应用该被如何组织，才能成功地交付给运维人员。这个问题跟如何写程序无关，它更关心如何获取面向（微）服务架构的程序组件，并简化围绕这类应用构建的工作流。
```

File: i18n/zh-Hans/1.purpose_and_goals.md [hunk 3]
```diff
@@ -1,3 +1,34 @@
...
+
+比如，时下典型的云上应用 可能由若干个微服务组成，每个微服务负责一个独立的逻辑模块，广义上我们把这种逻辑模块叫做『一个应用』（译者注：此处应用与前述云上应用不在一个维度，可以理解为子应用）。这些应用的整个声明周期中需要被配置、部署、授权认证、更新和删除。有时这些应用（指子应用）必须被整体对待，而有时又必须以不同的等级区别对待。最重要的是，经常这些应用并非被一个人或者一个团队来管理，而是由很多必须互相配合的团队来共同完成可靠、稳定和及时交付的目标。
```
Comment (wonderflow) [inline i18n/zh-Hans/1.purpose_and_goals.md]:
```suggestion
比如，时下典型的云上应用 可能由若干个微服务组成，每个微服务负责一个独立的逻辑模块，广义上我们把这种逻辑模块叫做『一个应用』。这些应用需要被配置、部署、审计、更新或删除。有时这些应用必须被当做一个整体对待，而有时又需要细粒度的区别对待。最重要的是，这些应用往往并非被一个人或者一个团队来管理，而是必须由多个团队相互配合，才能达成可靠、稳定、及时交付的目标。
```

File: i18n/zh-Hans/1.purpose_and_goals.md [hunk 4]
```diff
@@ -1,3 +1,34 @@
...
+
+规范提供了一种致力于分离开发和运维（运营）的应用描述。并且，规范为管理这种应用提供了建议的模式和处理过程。规范为云原生应用们，包括公有云技术栈、on-prem解决方案和IOT/边缘技术栈等 描述了一种模型。通过明确规定通用模型，规范为云原生应用共享描述（译者注：可理解为一个应用的oam描述文件，此处可解释为通过定义oam模型，为一个应用程序基于多种方式实现但又方便管理运营提供了基础）的多种实现提供了基础。
```
Comment (wonderflow) [inline i18n/zh-Hans/1.purpose_and_goals.md]:
```suggestion
本规范提供了一种致力于分离开发和运维两种角色的应用描述。此外，它也为管理这类应用提供了范式和处理方式。它为云原生应用描述了一种涵盖公有云、企业级解决方案和IoT/边缘等技术的模型。通过规定通用模型的方式，本规范为云原生应用的多种实现方式使用同一种描述提供了的基础。
```

File: i18n/zh-Hans/1.purpose_and_goals.md [hunk 5]
```diff
@@ -1,3 +1,34 @@
...
+
+## __开放规范定义了如下内容：__
+- __Component schematics__, 开发者在其中声明他们交付的代码的 _基础设施无关的_ 运维特性
+- __Application scopes__, 一种将Component分组为具有共同特性的松耦合应用的方法。
+- __Traits__ 指定操作特征给Component实例
+- __Application configuration__, 定义Component的deployment、Traits和application scopes
```
Comment (wonderflow) [inline i18n/zh-Hans/1.purpose_and_goals.md]:
```suggestion
- __Application configuration__, 定义了Component、Traits和application scopes 的组合运行方式。
```

File: i18n/zh-Hans/2.overview_and_terminology.md [hunk 6]
```diff
@@ -1,3 +1,268 @@
 # 概述和术语
 
-// TODO
\ No newline at end of file
+这部分提供了规范的概述和其中用到的技术术语的解释。开始部分明确区分了组织内部 与run起云原生应用有关的各种角色。然后，展开解读在本规范文档通篇会用到的特定技术术语。
```
Comment (wonderflow) [inline i18n/zh-Hans/2.overview_and_terminology.md]:
```suggestion
这部分提供了规范的概述和其中用到的技术术语的解释。它从明确运行云原生应用涉及到的组织内的不同角色开始， 然后，展开解读贯穿规范全篇的特定属于。
```

File: i18n/zh-Hans/2.overview_and_terminology.md [hunk 7]
```diff
@@ -1,3 +1,268 @@
...
+
+## 角色和职责
+
+作为规范的基础，需要定义如下角色。这些角色定义抽取于各种用户故事、调研和用户输入。尽管这些角色并不代表全部（有关角色），但是角色区分定义，是本规范的基本目标。
+
+- __开发者__ 以代码形式交付业务价值。同时他们应该理解所交付代码的运维特性，他们不关心运维需求如何被满足。比如，开发者可能清楚自己的代码往文件系统上某个特定路径写数据，但是不必关心何种vlolume或磁盘挂载到这个路径上，或者依赖是怎么被满足的。
```
Comment (wonderflow) [inline i18n/zh-Hans/2.overview_and_terminology.md]:
```suggestion
- __开发者__ 以代码形式交付业务价值。同时他们应该理解所交付代码的运维特性，他们不关心运维需求如何被满足。比如，开发者可能清楚自己的代码往文件系统上某个特定路径写数据，但是不必关心何种 volume 或磁盘挂载到这个路径上，或者依赖是怎么被满足的。
```

File: i18n/zh-Hans/2.overview_and_terminology.md [hunk 8]
```diff
@@ -1,3 +1,268 @@
...
+
+- __应用运维__ 通过配置、安装、管理组件和应用（比如更新、扩容、自动恢复等等）来传递业务价值。与开发者不同，应用运维关心的是组件和应用的运维需求如何被满足。比如，如果一个开发者声明组件写数据到文件系统的某个路径，应用运维人员需要将合适的volume挂载到这个path,来满足需求
+
+- __基础设施运维__ 通过管理低层次（底层）基础设施组件来体现价值。 其中，基础设施的范围可能包括从内部网络中管理物理硬件到直接管理公有云上的云服务等诸多维度. 基础设施运维人员 很少关注某个特定应用的配置需求，而是将工作专注在整个企业的基础设施管理全景。比如，基础设施运维人员可能管理用来持久化数据的底层存储系统
```
Comment (wonderflow) [inline i18n/zh-Hans/2.overview_and_terminology.md]:
```suggestion
- __基础设施运维__ 通过管理底层基础设施部分来体现价值。 其中，基础设施的范围可能包括从管理内网中的物理设备到直接管理公有云上的云服务等诸多维度。基础设施运维人员很少关注某个特定应用的配置需求，而更倾向于关注全局性的企业级基础设施该如何管理。比如，基础设施运维人员会管理用来提供持久化存储的底层系统。
```

File: i18n/zh-Hans/2.overview_and_terminology.md [hunk 9]
```diff
@@ -1,3 +1,268 @@
...
+
+在很多组织和机构，有些个人可能同时担任上述多种角色。同时，如上案例所述，这些角色经常一起工作以共同管理他们的企业级应用程序
```
Comment (wonderflow) [inline i18n/zh-Hans/2.overview_and_terminology.md]:
```suggestion
在很多组织和机构，有些个人可能同时担任上述多种角色。同时，就像上面提到的案例，这些角色也经常需要彼此协作以便共同管理企业级应用程序。
```

File: i18n/zh-Hans/2.overview_and_terminology.md [hunk 10]
```diff
@@ -1,3 +1,268 @@
...
+## 模型概述
+
+规范提倡如下定义云原生应用的模型：
+> 云原生应用是一组相互关联但又相互分离的 _组件_（服务、任务、工作器），当结合配置结合并在适当的运行时启动实例时，这些组件一起实现统一的功能目的。
```
Comment (wonderflow) [inline i18n/zh-Hans/2.overview_and_terminology.md]:
```suggestion

> 云原生应用是一组相互关联但又彼此解耦的 _组件_（服务、任务、运行器），当通过配置组合起来并在适当的运行时实例化以后，共同完成一个统一的功能性目标。
```

File: i18n/zh-Hans/2.overview_and_terminology.md [hunk 11]
```diff
@@ -1,3 +1,268 @@
...
+
+模型定义如下：
+
+ - _Components_ 代表一个可运行单元，同时附带schematic描述
```
Comment (wonderflow) [inline i18n/zh-Hans/2.overview_and_terminology.md]:
```suggestion
 - _Components_ 代表一个可运行单元，同时附带语法描述
```

File: i18n/zh-Hans/2.overview_and_terminology.md [hunk 12]
```diff
@@ -1,3 +1,268 @@
...
+ - _Workload types_ 明确区分了组件可被执行的不同负载类型
+ - _Traits_ 是用特定运维规范来增强Conponent后的的包装层，代表运维人员关注点，而非开发者关注点。
+ - _Application scopes_ 通过共同属性或依赖来分组以代表应用边界
+ - 一个 _application configuration_ 描述一套Component\Trait\Application scopes，在其中，上述内容被用配置参数和元数据来规制、组合
```
Comment (wonderflow) [inline i18n/zh-Hans/2.overview_and_terminology.md]:
```suggestion
 - 一个 _application configuration_ 描述了一个包含Component、Trait、Application scopes的集合该如何组织，以及如何绑定配置参数和元数据。
```

File: i18n/zh-Hans/2.overview_and_terminology.md [hunk 13]
```diff
@@ -1,3 +1,268 @@
...
+
+如此，一个 _应用_ 是一组组件的集合，其中每个组件附带一套集成到一个或一些应用边界的运维接口和能力
```
Comment (wonderflow) [inline i18n/zh-Hans/2.overview_and_terminology.md]:
```suggestion
所以，一个 _应用_ 包含了多个组件，每个组件附带一组运维特征，同时每个组件也都被囊括到了一个或多个应用边界中。
```

File: i18n/zh-Hans/2.overview_and_terminology.md (hunk 6 above)
Comment (KaiwenWang) [reply i18n/zh-Hans/2.overview_and_terminology.md]: 展开解读贯穿规范全篇的特定术语

**Comment by KaiwenWang** [comment]
PR updated according to the comments above.  

Sorry for long time no updating.

Files changed  in this PR shows the files have beed updated but commit do not show the latest.

Expecting more question and inspectiton.

**Comment by wonderflow** [comment]
@KaiwenWang  I notice you are working in progress, could you help translate the rest part of this PR.  I'll review again after you finish it, Thanks!

**Comment by KaiwenWang** [comment]
> @KaiwenWang I notice you are working in progress, could you help translate the rest part of this PR. I'll review again after you finish it, Thanks!

The rest of second part has been translated. Expect your  review

**Comment by hongchaodeng** [comment]
lgtm

**Pull request merged**